src/olympia/stats/views.py

```diff
@@ -162,6 +155,8 @@ def zip_overview(downloads, updates):
         start_date = downloads[0]['date']
     if updates:
         d = updates[0]['date']
+        if isinstance(d, str):
+            d = datetime.strptime(d, '%Y-%m-%d').date()
```

Comment:
is this a drive-by fix for something else?

Reply:
d'oh. I meant to comment on this line too, sorry. Yes. That's because BigQuery returns datetime instances but the test suite uses dates as strings. I feel like it is safer to have that. WDYT?

Reply:
I think we should be testing with the same format really, so the test data is datetime (at least by the time it's consumed by the view).  If this is going to be a big change feel free to put it in a follow-up.

Reply:
You're right. I updated the test file with `date` instances. I also updated the BigQuery function to format series so that it returns `date` instances too. It should be the case but better safe than sorry.